### DISCUSSION TITLE
HW 5 - Part 2, #1

### DISCUSSION THREAD

**Discussion by RebeccaVillanueva** [cooperrc/computational-mechanics#135] [General]
For 2.1 should the temps range from 100 (Tbase) to 25 (Tsink)? I feel like this is obviously yes, but my T6 is arounf 45 degC and I can't figure out why

**Comment by cooperrc** [comment]
Yes, you have to change the final finite difference equation such that $T_6=25^oC$. Otherwise, you're using the same boundary conditions as the reading.

**Reply by RebeccaVillanueva** [reply]
Okay, that's what I thought but I can't figure out where to factor in Tsink. I think the diagonal factor is the same and the A and b equations. Does it have something to do with A[-1,-1] and b[-1]?

**Reply by RebeccaVillanueva** [reply]
Never mind, I figured it out :)
